subsys/event_manager/event_manager.c

```diff
@@ -30,34 +31,42 @@ static bool log_is_event_displayed(const struct event_type *et)
 	return atomic_test_bit(_event_manager_event_display_bm.flags, idx);
 }
 
+static void log_event_using_buffer(const struct event_header *eh, const struct event_type *et)
+{
+	char log_buf[CONFIG_EVENT_MANAGER_EVENT_LOG_BUF_LEN];
```

Comment:
build assert that CONFIG_EVENT_MANAGER_EVENT_LOG_BUF_LEN is greater or equal 2

Reply:
It could be, but we also need to allow to use CONFIG_EVENT_MANAGER_EVENT_LOG_BUF_LEN  == 0